### ISSUE TITLE
Stack import diagrams

### ISSUE THREAD

**Issue by matzebond** [diagrams/diagrams#17]
When using stack 
```
dependencies:
 - diagrams
```
 in the `package.yaml` doesn't work and one has to depend on every diagrams
```
dependencies:
- diagrams
- diagrams-lib
- diagrams-contrib
- diagrams-svg
- diagrams-core
```
(thanks https://github.com/commercialhaskell/stack/issues/3964 for the fix)

Is there a way to improve this situation on the diagrams side?
Mentioning this in the tutorial would also be nice.

**Comment by byorgey** [comment]
No, there is no way to improve this situation; the `diagrams` package exists only as a convenience which depends on the other packages, and by design `cabal` and `stack` do not consider transitive dependencies when resolving module imports.  However, I understand why you would find it confusing and I agree it should be mentioned somewhere in the docs.

I also note that you should definitely be able to remove `diagrams` from your list of dependencies; and you can probably remove `diagrams-core` as well, depending on what you are doing.
